### PR TITLE
Updates styleguidist to v9

### PR DESCRIPTION
**Pull request by KaiVolland**
This updates stylegudist to version 9.
It also modifies the examples as needed and adds missing packages.

@terrestris/devs please review

closes #866
closes #847

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -47,5 +48,11 @@ module.exports = {
       name: 'UploadButton',
       components: 'src/Component/UploadButton/**/*.tsx'
     }]
-  }]
+  }],
+  // moduleAliases: {
+  //   'antd': path.resolve(__dirname, 'node_modules/antd')
+  // },
```
Comment (dnlkoch) [inline styleguide.config.js]: Can be removed.

File: package.json [hunk 2]
```diff
@@ -92,6 +92,7 @@
     "case-sensitive-paths-webpack-plugin": "2.2.0",
     "copyfiles": "2.1.0",
     "coveralls": "3.0.3",
+    "css-loader": "2.1.0",
```
Comment (dnlkoch) [inline package.json]: Is this really needed for the antd css require in `webpack.config.json`?

File: package.json [hunk 3]
```diff
@@ -108,9 +109,10 @@
     "react": "16.8.3",
     "react-docgen-typescript": "1.12.3",
     "react-dom": "16.8.3",
-    "react-styleguidist": "8.0.6",
+    "react-styleguidist": "9.0.1",
     "rimraf": "2.6.3",
     "source-map-loader": "0.2.4",
+    "style-loader": "0.23.1",
```
Comment (dnlkoch) [inline package.json]: See comment above.

File: src/Component/RuleGenerator/RuleGenerator.example.md [hunk 4]
```diff
@@ -1,8 +1,7 @@
 This demonstrates the use of `RuleGenerator`.
 
 ```jsx
-const React = require('react');
-const { RuleGenerator } = require('../../index');
```
Comment (dnlkoch) [inline src/Component/RuleGenerator/RuleGenerator.example.md]: Even if not needed for styleguide itself, I would suggest to import all related classes for better real world appliance of the examples.

Comment (KaiVolland) [reply src/Component/RuleGenerator/RuleGenerator.example.md]: I had to remove it, as i got 'already defined' errors when leaving the import in the example.

File: package.json (hunk 2 above)
Comment (KaiVolland) [reply package.json]: Yes. 'module not found' errors where thrown before.

File: package.json (hunk 3 above)
Comment (KaiVolland) [reply package.json]: Yes. 'module not found' errors where thrown before.

**Pull request merged**